crates/bevy_input/src/input.rs

```diff
@@ -1,7 +1,33 @@
 use bevy_utils::HashSet;
 use std::hash::Hash;
 
-/// A "press-able" input of type `T`
+// unused import, but needed for intra doc link to work
+#[allow(unused_imports)]
+use bevy_ecs::schedule::State;
+
+/// A "press-able" input of type `T`.
+///
+/// This type can be used as a resource to keep the current state of an input, by reacting to
+/// events from the input. For a given input value:
+///
+/// * [`Input::pressed`] will return `true` between a press and a release event.
+/// * [`Input::just_pressed`] will return `true` for one frame after a press event.
+/// * [`Input::just_released`] will return `true` for one frame after a release event.
+///
+/// In case multiple systems are checking for [`Input::just_pressed`] or [`Input::just_released`]
+/// but only one should react, for example in the case of triggering
+/// [`State`] change, you should consider clearing the input state, either by:
+///
+/// * Using [`Input::just_pressed_and_clear`] or [`Input::just_released_and_clear`] instead.
```

Comment:
I wanted to note that these no longer exist with the changes that @cart pushed to this PR (`just_pressed_and_clear` -> `clear_just_pressed`).

It might be a good idea to enable [rustdoc linting](https://doc.rust-lang.org/rustdoc/lints.html#broken_intra_doc_links) on the CI to catch issues like there?

Reply:
fixed those two and another one in #2007 

I checked on how to enable rustdoc linting cleanly for our CI, `cargo` is missing a feature for that, will try to get it. We're also trying to get global dead links check with #1590 but too many errors for now